src/engine/source/vdscanner/qa/test_false_negative_data/045/Readme.md

```diff
@@ -0,0 +1,16 @@
+# Description
+
+Vulnerability detection validation for `perl-modules` and `linux-hwe-6.8` on Ubuntu.
+
+# Platforms
+
+## Windows
```

Comment:
```suggestion
## Ubuntu
```
...but since the input events refer to two different version of Ubuntu we can arrange them like in this case: https://github.com/wazuh/wazuh/blob/97d62024fcfb2c625101349f6cb564a3258e18db/src/engine/source/vdscanner/qa/test_false_negative_data/024/Readme.md

Reply:
Done.